test/e2e/storage/regional_pd.go

```diff
@@ -342,8 +341,7 @@ func testRegionalAllowedTopologies(c clientset.Interface, ns string) {
 			"type":             "pd-standard",
 			"replication-type": "regional-pd",
 		},
-		ClaimSize:    "2Gi",
-		ExpectedSize: "2Gi",
```

Comment:
why do we no longer care about expected size? Also I always forget but is "200Gi" in this case the same as the "200Gb" minimum. Or is Gi smaller/bigger

Reply:
oh I got mislead because I saw some test cases not pass in ExpectedSize so I thought it might be defaulting if it was unset.  I added it back.

GCE PD GB == K8s Gi

From  https://cloud.google.com/compute/pricing#disk/:
```
Disk size, machine type memory, and network usage are calculated in gigabytes (GB), where 1 GB is 2^30 bytes. This unit of measurement is also known as a gibibyte (GiB).
```